.github/.jira_sync_config.yaml

```diff
@@ -0,0 +1,21 @@
+# Sync GitHub issues to Jira issues
+
+# Configuration syntax:
+# https://github.com/canonical/gh-jira-sync-bot/blob/main/README.md#client-side-configuration
+settings:
+  # Repository specific settings
+  components: # Jira components that will be added to Jira issue
+    - mysql-vm
```

Comment:
Does it make sense to use this component?  My understanding is that sysbench can be integrated with both postgresql and mysql (both vm & k8s).

Reply:
today it is a common charm for MySQL and PostgreSQL. common-libs OR mysql-vm/k8s+postgresql-vm/k8s...